cmd/gon/main.go

```diff
@@ -140,6 +140,46 @@ func realMain() int {
 		}
 	}
 
+	// If not specified in the configuration, we initialize a new struct that we'll
+	// load with values from the environment.
+	if cfg.AppleId == nil {
+		cfg.AppleId = &config.AppleId{}
+	}
+
+	if cfg.AppleId.Username == "" {
+		appleIdUsername, ok := os.LookupEnv("AC_USERNAME")
+
+		if ok {
```

Comment:
This is fine, but I prefer to do a `!ok` check and exit, so that we don't need a full `else` case.